VideoWeb/VideoWeb/Controllers/ParticipantsController.cs

```diff
@@ -175,7 +175,7 @@ public async Task<IActionResult> GetParticipantsWithContactDetailsByConferenceId
             }
             try
             {
-                var conference = await conferenceService.GetConference(conferenceId);
+                var conference = await conferenceService.ForceGetConference(conferenceId);
```

Comment:
This endpoint is the one used in the command centre by the vhofficer? Does this need to be updated aswell

Reply:
good point, thanks. Have reverted the change